Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -264,26 +264,30 @@ public class Teacher
         public readonly string lastName;
         public string department;
         public readonly DateTime? birthDate;
+        public Car car;
 
         public Teacher()
         {
             this.firstName = null;
             this.lastName = null;
             this.department = null;
             this.birthDate = null;
+            this.car = null;
         }
         
-        public Teacher(string firstName, string lastName, string department, DateTime birthDate)
+        public Teacher(string firstName, string lastName, string department, DateTime birthDate, Car car)
```

Comment:
These objects should be purpose-made for the tests they cover. I think by doing this these objects are going to get very complex. Rather make a new one.

Reply:
Addressed.